delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala

```diff
@@ -171,6 +171,22 @@ final class ResourcesRoutes(
                           }
                         )
                       },
+                      (pathPrefix("updateSchema") & put & pathEndOrSingleSlash) {
+                        authorizeFor(ref, Write).apply {
+
+                          emit(
+                            IO.fromOption(schemaOpt)(IdenticalSchema())
```

Comment:
I would say the error here is that the schema is not specified rather than it being the same

Reply:
My thinking is that when updating resources `_` means keep the same schema. I am reworking things a bit anyway

Reply:
I changed the rejection name

Reply:
Resolving this since the name changed